test/functionalTest/cases/117_convop_using_standard_ops/postAllEntitiesWithTypeAndId.test

```diff
@@ -672,4 +672,4 @@ Date: REGEX(.*)
 
 --TEARDOWN--
 brokerStop CB
-# dbDrop CB
+dbDrop CB
```

Comment:
Maybe is a goog momento to do some 'grep dbDrop | grep #' in all the .test... just to be sure all them are fixed ;)